package.json

```diff
@@ -54,17 +49,17 @@
   },
   "devDependencies": {
     "@rgbboy/testcheck": "^1.0.0-rc.3",
-    "babel-eslint": "^8.2.1",
-    "babel-jest": "22.2.2",
+    "babel-eslint": "^8.2.5",
+    "babel-jest": "23.2.0",
     "babel-preset-react-native": "4.0.0",
     "bugsnag-sourcemaps": "^1.0.3",
-    "commander": "^2.13.0",
-    "contentful-management": "^4.1.4",
-    "detox": "^7.3.3",
-    "dotenv": "^5.0.0",
+    "commander": "^2.16.0",
+    "contentful-management": "^5.1.4",
+    "detox": "^8.0.0",
```

Comment:
I was planning on updating detox in its own PR as there some cool new stuff to reduce boilerplate in our tests

Reply:
Ah nice. I’ll remove detox from this one then.